### PR TITLE
fix: correct spacing in the pretty printer

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

File: src/Lean/Parser/Command.lean [hunk 1]
```diff
@@ -79,7 +80,7 @@ def structureTk          := leading_parser "structure "
 def classTk              := leading_parser "class "
 def «extends»            := leading_parser " extends " >> sepBy1 termParser ", "
 def «structure»          := leading_parser
-    (structureTk <|> classTk) >> declId >> many Term.bracketedBinder >> optional «extends» >> Term.optType
+    (structureTk <|> classTk) >> declId >> many (ppSpace >> Term.bracketedBinder) >> optional «extends» >> Term.optType
```
Comment (semorrison) [inline src/Lean/Parser/Command.lean:83]: See https://github.com/leanprover-community/mathlib3port/blob/9f61de08d4e62b7d0efe387e65b0af62f1a08d8a/Mathbin/Combinatorics/Quiver.lean#L47 for the problem being fixed here. (Note that all files in the mathlib3port repo are generated by `mathport`, and have come through the pretty printer.)

Comment (semorrison) [reply src/Lean/Parser/Command.lean:83]: Adding `ppSpace` here follows the example in `declSig`. (Thanks @digama0 for explaining this to me.)

File: src/Lean/Parser/Command.lean [hunk 2]
```diff
@@ -88,7 +89,7 @@ declModifiers false >> («abbrev» <|> «def» <|> «theorem» <|> «constant»
 @[builtinCommandParser] def «section»      := leading_parser "section " >> optional ident
 @[builtinCommandParser] def «namespace»    := leading_parser "namespace " >> ident
 @[builtinCommandParser] def «end»          := leading_parser "end " >> optional ident
-@[builtinCommandParser] def «variable»     := leading_parser "variable" >> many1 Term.bracketedBinder
+@[builtinCommandParser] def «variable»     := leading_parser "variable" >> many1 (ppSpace >> Term.bracketedBinder)
```
Comment (semorrison) [inline src/Lean/Parser/Command.lean:92]: An [example](https://github.com/leanprover-community/mathlib3port/blob/9f61de08d4e62b7d0efe387e65b0af62f1a08d8a/Mathbin/Combinatorics/Quiver.lean#L177).

**Comment by Kha** [comment]
Thanks!

**Pull request merged**